ipi/engine/properties.py

```diff
@@ -230,8 +230,8 @@ class Properties:
     """
 
     _DEFAULT_FINDIFF = 1e-4
-    _DEFAULT_FDERROR = 1e-6
-    _DEFAULT_MINFID = 1e-7
+    _DEFAULT_FDERROR = 1e-5
+    _DEFAULT_MINFID = 1e-8
```

Comment:
All changes look good to me, but what is the driving force behind adjusting the defaults here?

Reply:
Boh, I played around a bit with these defaults to try and find stabler ones. Not sure it helped TBH. Main realization is that you need VERY converged PPPM electrostatics to have stable finite difference calculations, but that's not i-PI's problem. Discuss this a bit in https://lab-cosmo.github.io/atomistic-cookbook/latest/examples/heat-capacity/heat-capacity.html

Reply:
ok